src/Idris/Driver.idr

```diff
@@ -184,7 +184,7 @@ stMain cgs opts
                                   pure res
 
                  doRepl <- catch (postOptions result opts)
-                                 (\err => emitError err *> pure False)
+                                 (\err => False <$ showTimeRecord <* emitError err)
```

Comment:
Because we continue on with False instead of quitting immediately, `showTimeRecord` will be done twice with this change as it's done at line 211. We can quit here or let the later showTimeRecord do the work

I think if I ever revisit this section this if-chain will be refactored into a sum type, it wouldn't save any space but should make what the ifs branches are doing more direct. Or maybe by then we'll have multi-way-if.

Reply:
Oh shoot you had already pushed a fix? I thought the PR was going stale and
needed just one small change to be mergeable. That's why I hate force pushes:
we completely lose the information of what happened when and what the new
changes are.

Reply:
I apologize, forgot the basic rule of don't force-push things other people can touch.